packages/fast-components-react-msft/src/action-toggle/action-toggle.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Action toggle",
+    "description": "An action toggle component's schema definition.",
+    "type": "object",
+    "id": "action-toggle",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "selected": {
+            "title": "Selected",
+            "type": "boolean"
+        },
+        "selectedContent": {
+            "title": "Selected text",
+            "type": "object"
+        },
+        "unselectedContent": {
+            "title": "Unselected text",
+            "type": "object"
+        },
+        "selectedLabel": {
+            "title": "Selected label",
```

Comment:
If these are aria, the title should say "Selected HTML aria-label attribute". If these are just the display string, then this title works.

Reply:
changed